core/src/smartcontracts/isi/asset.rs

```diff
@@ -318,15 +365,6 @@ pub mod isi {
         match definition.mintable() {
```

Comment:
At this point, we could revert back to a boolean. But I would recommend keeping the `Mintable::Once` variant. We want to offer people the choice to register an asset definition but to defer the minting.

Reply:
You can still register _asset definition_ without minting.
After this PR `Mintable::Not` behaves essentially the same as `Mintable::Once` currently, since you can register _asset to an account_ with whatever value, that's why I removed it.

> At this point, we could revert back to a boolean

Didn't do it based on [this](https://github.com/hyperledger/iroha/blob/916a2cc584ba110775eafeae69e9658157f704c9/data_model/src/asset.rs#L191)